modules/models.py

```diff
@@ -2805,6 +2849,21 @@ class GameSide(BaseModel):
     win_confirmed = BooleanField(default=False)
     team_chan_external_server = BitField(unique=False, null=True, default=None)
 
+    def as_json(self) -> dict[str, Any]:
```

Comment:
and here

Reply:
This is a feature of Python 3.9, see [PEP 585](https://www.python.org/dev/peps/pep-0585/). If compatibility with 3.7 and 3.8 is required, `from __future__ import annotations` can be used.